README.md

```diff
@@ -11,41 +11,21 @@
 ---
 <!-- /ix-docs-ignore -->
 
-- [Dependencies](#dependencies)
-- [Installation](#installation)
-  - [Standalone](#standalone)
-  - [Using Composer](#using-composer)
 - [Usage](#usage)
 - [Signed URLs](#signed-urls)
 - [Srcset Generation](#srcset-generation)
-
-## Dependencies
-
-The tests have a few external dependencies. To install those:
-
-```bash
-phpunit --bootstrap src/autoload.php tests/
-```
-
-## Installation
-
-### Standalone
-
-Just copy the files to your project, and include the `src/autoload.php` file. We recommend using Composer if at all possible.
-
-### Using Composer
-
-Define the following requirement in your `composer.json` file:
-
-```json
-{
-  "require": {
-    "imgix/imgix-php": "dev-master"
-  }
-}
-```
-
-And include the global `vendor/autoload.php` autoloader.
+  - [Fixed-Width Images](#fixed-width-images)
+    - [Variable Quality](#variable-quality)
+  - [Fluid-Width Images](#fluid-width-images)
+    - [Custom Widths](#custom-widths)
+    - [Width Ranges](#width-ranges)
+    - [Width Tolerance](#width-tolerance)
+- [The `ixlib` Parameter](#the-ixlib-parameter)
+- [Installation](#installation)
+  - [Standalone](#standalone)
+  - [Using Composer](#using-composer)
```

Comment:
Looks like the `Installation` section got accidentally moved towards the bottom.

Reply:
Yeah of course!